src/index.html

```diff
@@ -2,17 +2,64 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
+
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+
     <title>Product cards</title>
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+
+  <body
+    class="card"
+    data-qa="card"
+  >
+    <div class="card__image"></div>
+
+    <div class="card__model">
+      APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+    </div>
+
+    <div class="card__product-code">Product code: 195434</div>
+
+    <div class="card__grade">
+      <div class="card__grade-stars"></div>
+
+      <div class="card__grade-reviews">Reviews: 5</div>
+    </div>
+
+    <div class="card__price"></div>
```

Comment:
The `.card__price` div is empty. If it's supposed to contain a price, it should include the price inside, and use a `span` or `p` tag with a class to style it.